RELEASE.md

```diff
@@ -0,0 +1,22 @@
+# Releasing the scorecard GitHub Action
+
+This is a draft document to describe the release process for the scorecard
+GitHub Action.
+
+(If there are improvements you'd like to see, please comment on the
+[tracking issue](https://github.com/ossf/scorecard-action/issues/33) or issue a
+pull request to discuss.)
+
+- [steps:](#steps)
+
+## steps:
+
+1. cut a scorecard release and wait for a container image to be created and tagged with new release. Note the hash of the container as `CH1`. Note: we do not need a scorecard release, we can use any stable version we want.
+2. update the hash pin in our [dockerfile](https://github.com/ossf/scorecard-action/blob/main/.github/Dockerfile#L26) to use the container hash `CH1` from step 1. Once the PR is merged, note the GitHub's commit hash as `GH2`.
+~~3. [manually trigger](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) the workflow to generate our container image. Note the hash of the container image generated as `CH3`. It can be found [here](https://github.com/ossf/scorecard-action/pkgs/container/scorecard-action) using the manifest's "digest"`.~~
```

Comment:
step 3 and 4 are remnants of the past, they are no longer needed.

Reply:
Removed!